docs/contribute/development/development_environment.rst

```diff
@@ -148,3 +148,13 @@ IDE Guides
 
    ide/vscode
    ide/clion
+
+WSL2 Settings
+=============
+If you are using a Linux box running on WSL2, the following minimal settings in your .wslconfig file are recommended:
```

Comment:
Maybe we can rephrase this and merge, so we don't need to wait for @MaximilianSoerenPollak? Currently it's difficult to tell when out-of-memory happens, as no-one has complained yet. I assume memory becomes an issue when you run an (old) C++ linker?!

```suggestion
If you encounter memory issues on WSL2, the following minimal settings in your .wslconfig file are recommended:
```